recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
```

Comment:
I put 2 Perl lib ( Switch.pm and Bio/Seq.pm ) in the my repo. The file use many relative path and when building conda recipe I had to change some. Furthermore the link above only have the bin code while the version which I used for Galaxy has snmf.sh and Snmf.pl to control the bin code after install.